packages/visx-demo/src/sandboxes/visx-radial-bars/Example.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useMemo, useState } from 'react';
+import { Arc } from '@visx/shape';
+import { Group } from '@visx/group';
+import { GradientLightgreenGreen } from '@visx/gradient';
+import { scaleBand, scaleRadial } from '@visx/scale';
+import { Text } from '@visx/text';
+import letterFrequency, { LetterFrequency } from '@visx/mock-data/src/mocks/letterFrequency';
+
+const data = letterFrequency;
+
+const getLetter = (d: LetterFrequency) => d.letter;
+const getLetterFrequency = (d: LetterFrequency) => Number(d.frequency) * 100;
+
+const frequencySort = (a: LetterFrequency, b: LetterFrequency) => b.frequency - a.frequency;
+const alphabeticalSort = (a: LetterFrequency, b: LetterFrequency) =>
+  a.letter.localeCompare(b.letter);
+
+const toRadians = (x: number) => (x * Math.PI) / 180;
+const toDegrees = (x: number) => (x * 180) / Math.PI;
+
+const barColor = '#93F9B9';
+const margin = { top: 20, bottom: 20 };
+
+export type RadialBarsProps = {
+  width: number;
+  height: number;
+  showControls?: boolean;
+};
+
+export default function Example({ width, height, showControls = true }: RadialBarsProps) {
+  const [rotation, setRotation] = useState(0);
+  const [sortAlphabetically, setSortAlphabetically] = useState(true);
+
+  // bounds
+  const xMax = width;
+  const yMax = height - margin.top - margin.bottom;
+  const radiusMax = Math.min(xMax, yMax) / 2;
+
+  const innerRadius = radiusMax / 3;
+
+  const xDomain = useMemo(
+    () => data.sort(sortAlphabetically ? alphabeticalSort : frequencySort).map(getLetter),
+    [sortAlphabetically],
+  );
+
+  const xScale = useMemo(
+    () =>
+      scaleBand<string>({
+        range: [0 + rotation, 2 * Math.PI + rotation],
+        domain: xDomain,
+        padding: 0.2,
+      }),
+    [rotation, xDomain],
+  );
+
+  const yScale = useMemo(
+    () =>
+      scaleRadial<number>({
+        range: [innerRadius, radiusMax],
+        domain: [0, Math.max(...data.map(getLetterFrequency))],
+      }),
+    [innerRadius, radiusMax],
+  );
+
+  return width < 10 ? null : (
+    <>
+      <svg width={width} height={height}>
+        <GradientLightgreenGreen id="green" />
```

Comment:
this `id` is probably okay, on the gallery page with all examples, this should be unique or could potentially conflict (if I see anything can easily fix)